modules/rmarkdown/parametrize.nf

```diff
@@ -0,0 +1,23 @@
+@Grab(group='org.yaml', module='snakeyaml', version='1.28')
```

Comment:
I use the external dependency snakeyaml for dumping the notebook parameters. 
Grab automatically fetches the dependency, but this obviously won't work on offline clusters. 

Possible solutions
 * Bundle the `jar` with the module somehow
 * Re-implement a simplistic yaml dumper in groovy 
 * make `nf-core download` fetch `@Grab` jars automatically